doc/releases/migration-guide-4.3.rst

```diff
@@ -64,6 +64,11 @@ Ethernet
 Networking
 **********
 
+* The HTTP server now respects the configured ``_config`` value. Check that
```

Comment:
I am baffled, I thought that one is suppose to add the needed entries between start and stop directives. @pdgendt, is the usage documented somewhere?

Reply:
The `re(^\w)` makes it so that the compliance check only verifies that lines starting with a word are sorted. So these bullet points would be skipped anyway.

Documentation is lacking, yes.